generation/internal/fallback.go

```diff
@@ -59,34 +59,3 @@ var stubDeployJob = Job{
 	},
 	Type: DeployJob,
 }
-
-var fallbackConfig = config.Config{
```

Comment:
created a function to generate the fallback config since it makes it easier to modify the jobs ( in this case adding a deploy job with the cicd steps)